src/ParseServer.js

```diff
@@ -210,7 +211,16 @@ class ParseServer {
 
     Config.validate(AppCache.get(appId));
     this.config = AppCache.get(appId);
-    hooksController.load();
+    if (hooksRefreshIntervalMs > 0) {
+      (function repeat() {
+        setTimeout(() => {
```

Comment:
I think we need to unref() the timer so it doesn't hold parse-server open if it's the only timer left in the event loop.